code/game/gamemodes/game_mode.dm

```diff
@@ -54,6 +55,8 @@
 	if(!Debug2)
 		if(playerC < required_players)
 			return 0
+		if(playerC > max_players)

```

Comment:
Recommend changing this to `if(playerC > max_players && max_players > 0)`. It would allow infinite players by setting `max_players` to 0 instead of setting an arbitrary limit of 500.